presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorQueryLogging.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor;
+
+import com.facebook.presto.execution.QueryManager;
+import com.facebook.presto.tests.DistributedQueryRunner;
+
+import com.google.common.collect.ImmutableMap;
+
+import org.testng.annotations.Test;
+
+import static java.lang.String.format;
+
+public class TestRaptorQueryLogging
+{
+    @Test
+    public void testQueryLoggingCount()
+            throws Exception
+    {
+        try (DistributedQueryRunner queryRunner = RaptorQueryRunner.createRaptorQueryRunner(ImmutableMap.of(), false, false)) {
+            queryRunner.execute("CREATE TABLE foo (x bigint)");
+            queryRunner.execute("SELECT * FROM foo");
+            queryRunner.execute("DROP TABLE foo");
+            try {
```

Comment:
Remove this try/catch. It shouldn't throw, and if it does, we definitely do not want to ignore it.